docs/en/observability/profiling-add-symbols.asciidoc

```diff
@@ -0,0 +1,151 @@
+[[profiling-add-symbols]]
+= Add symbols for native frames
+
+beta::[]
+
+To see function names and line numbers in traces of applications written in programming languages that 
+compile to native code (C, C++, Rust, Go, etc.), you need to push symbols to the cluster.
+
+You can do this with the `elastic-profiling` CLI utility.
+
+Download the `elastic-profiling` binary https://releases.prodfiler.com/elastic-profiling_linux_amd64.tar.gz[here].
+
+NOTE: As the `elastic-profiling` binary is currently only distributed for `Linux x86_64`, you need a Linux machine
+to use it.
+
+[discrete]
+[[profiling-use-elastic-profiling]]
+== How to use the `elastic-profiling` binary
+
+Before using the `elastic-profiling` binary, you need the following information for the deployment where you're running Universal Profiling:
+
+* Credentials (either an API key or username and password) for the `superuser` Elasticsearch role (typically, the `elastic` user).
+* Copy the deployment's Cloud ID from the deployment overview page.
++
+[role="screenshot"]
+image::images/profiling-cloud-id.png[]
+* Copy the deployment's APM Component ID from the deployment overview page.
++
+[role="screenshot"]
+image::images/profiling-apm-cluster-id.png[]

```

Comment:
I'm realizing now this image is out-of-date with the current UI of Cloud.

Here's an updated version.

![Screenshot from 2023-03-23 15-48-39](https://user-images.githubusercontent.com/4930991/227243166-04e5b064-a427-426a-b220-a59cf8e30fd0.png)